src/siwe.ts

```diff
@@ -86,6 +88,24 @@ export class SiweMessage {
     }
   }
 
+  static fromCacao(cacao: Cacao): SiweMessage {
+    const account = AccountId.parse(cacao.p.iss.replace('did:pkh:', ''))
+    return new SiweMessage({
+      domain: cacao.p.domain,
+      address: account.address,
+      statement: cacao.p.statement,
+      uri: cacao.p.aud,
+      version: cacao.p.version,
+      nonce: cacao.p.nonce,
+      issuedAt: new Date(cacao.p.iat).toISOString(),
+      expirationTime: cacao.p.exp ? new Date(cacao.p.exp).toISOString() : undefined,
+      notBefore: cacao.p.nbf ? new Date(cacao.p.nbf).toISOString() : undefined,
```

Comment:
Careful, `Date` uses ms not seconds.

Reply:
The CAIP issue linked below suggests that ISO8601 does support ms, but we need to switch to the string repr due to timezone identifiers, so im guessing this comment is irrelevant now?